### PR TITLE
Add support for PLF 2nd part

### PR DESCRIPTION
**Pull request by ronnix**
capture d ecran 2018-10-29 a 20 14 43

### CODE REVIEW

File: repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py [hunk 1]
```diff
@@ -108,9 +111,16 @@ def gen_lectures(acte: dict, textes: Dict[str, Texte]) -> Iterator[Lecture]:
         assert result.texte is not None
         texte = textes[result.texte]
 
-        yield Lecture(  # type: ignore
-            chambre=chambre, titre=titre, texte=texte, organe=result.organe
-        )
+        parties: List[Optional[int]] = [1, 2] if is_plf else [None]
+
+        for partie in parties:
+            yield Lecture(  # type: ignore
```
Comment (davidbgk) [inline repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py]: I wonder if we should uniformize creations with the `model.create()` method. 🤔

Comment (ronnix) [reply repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py]: Here it’s not an SQLAlchemy model, it’s a dataclass

Comment (davidbgk) [reply repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py]: Indeed 🤦‍♂️

Comment (ronnix) [reply repondeur/zam_repondeur/fetch/an/dossiers/dossiers_legislatifs.py]: I got confused too when looking bad at that code

**Pull request merged**